File: Solution/LCP01.猜数字/LCP01.猜数字.cpp
/*
 * @Descroption: LCP 01. 猜数字
 * @Author: EmoryHuang
 * @Date: 2021-07-08 18:43:43
 * @解题思路:
 * 简单题，遍历一遍
 */

class Solution {
   public:
    int game(vector<int>& guess, vector<int>& answer) {
        int ans = 0;
        for (int i = 0; i < 3; i++)
            if (guess[i] == answer[i]) ans++;
        return ans;
    }
};

File: Solution/LCP01.猜数字/LCP01.猜数字.py
'''
Descroption: LCP 01. 猜数字
Author: EmoryHuang
Date: 2021-07-08 18:42:37
解题思路:
简单题，遍历一遍
'''


class Solution:
    def game(self, guess: List[int], answer: List[int]) -> int:
        ans = 0
        for i in range(3):
            if guess[i] == answer[i]:
                ans += 1
        return ans
